src/node.cc

```diff
@@ -3901,9 +3906,7 @@ Environment* CreateEnvironment(Isolate* isolate,
       HandleCleanup,
       nullptr);
 
-  if (v8_is_profiling) {
-    StartProfilerIdleNotifier(env);
-  }
+  StartProfilerIdleNotifier(env);
```

Comment:
I don't think this is an acceptable change.  Starting idle notification unconditionally has a performance impact, that's why it's only done when profiling is enabled.  You should handle this in v8-profiler.